app/uk/gov/hmrc/gform/des/DesConnector.scala

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2017 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.gform.des
+
+import com.typesafe.config.Config
+import play.api.libs.json.{ JsValue, Json, OFormat }
+import uk.gov.hmrc.gform.wshttp.WSHttp
+import uk.gov.hmrc.play.http.logging.Authorization
+import uk.gov.hmrc.play.http.{ HeaderCarrier, NotFoundException }
+
+import scala.concurrent.{ ExecutionContext, Future }
+
+class DesConnector(wSHttp: WSHttp, baseUrl: String, desConfig: Config) { //TODO make the connector easy to move from service to service
```

Comment:
1. How about passing instance of DesConnectorConfig here instead of typesafe config? Typesafe config might or might not have all required fields.

```
case class DesConnectorConfig(baseUrl: String, basePath: String, token: String, environment: String) 
```

2. Since it's in proper package is easily removable, so we don't need this comment do we? `//TODO make the connector easy to move from service to service`